### PR TITLE
feat(TMRX-1543): Lead story 3 refactor

### PR DESCRIPTION
**Pull request by milanharia-tm**
### Description

Refactors lead story 3 to no longer use the `useBreakpoint` hook.
Removes stand first from the third story in the left column when rendered at medium breakpoint.

[JIRA-1543](https://nidigitalsolutions.jira.com/browse/TMRX-1543)

### Checklist

- [X] Have you done any manual testing?
- [ ] Does it have automated tests?
- [ ] Do you need any other PRs merged before this (if so please list)?
- [ ] Do you need to update the README/Runbook
- [ ] Have you checked for [accessibility](https://nidigitalsolutions.jira.com/wiki/spaces/TNLDigital/pages/3898048523/Accessibility+Dev+Guide#Checklist)?

### Screenshots (if appropriate):

**Before**

Screenshot 2023-10-03 at 15 27 55

**After**

Screenshot 2023-10-03 at 15 27 30

### CODE REVIEW

File: packages/ts-newskit/src/slices/shared/article-stacks.tsx [hunk 1]
```diff
@@ -29,117 +42,137 @@ export const ArticleStackLarge = ({
...
+            {articleIndex < articleArr.length - 1 && (
+              <Visible md lg xl>
+                <Divider
+                  overrides={{
+                    stylePreset: 'lightDivider'
+                  }}
+                  vertical
+                />
+              </Visible>
+            )}
```
Comment (yukiinasawa) [inline packages/ts-newskit/src/slices/shared/article-stacks.tsx:156]: Just wonder as this is a refactor ticket - `ArticleStackSmall` and `ArticleStackLarge` are quite repetitive, could we combine those into one? Or are we keeping them separate on purpose?

File: packages/ts-newskit/src/slices/lead-story-3/article-stack.tsx [hunk 2]
```diff
@@ -1,67 +1,94 @@
...
+                  ...article,
                   headlineTypographyPreset: 'editorialHeadline020',
-                  hideImage: true
+                  hideImage: true,
+                  hasTopBorder: false
                 };
+
+          if (index === 0) {
+            return (
+              <>
```
Comment (ibrahim-kurhan) [inline packages/ts-newskit/src/slices/lead-story-3/article-stack.tsx:43]: do we need this `<></>`

File: packages/ts-newskit/src/slices/shared/article-stacks.tsx (hunk 1 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/slices/shared/article-stacks.tsx:153]: yes thats was the idea just to remove the complexity from a component and split it into 2

**Review by ibrahim-kurhan — APPROVED** [review]

File: packages/ts-newskit/src/components/misc/update-button/__tests__/update-button-with-delay.test.tsx [hunk 3]
```diff
@@ -46,4 +46,22 @@ describe('Render UpdateButtonWithDelay', () => {
...
+      'Update Now',
+      handleClickMock,
+      true,
+      '2023-10-12T00:00:00.000Z',
+      '12345',
+      false
+    );
+
+    jest.advanceTimersByTime(120000);
+    expect(true).toBe(true);
```
Comment (bibicollins1) [inline packages/ts-newskit/src/components/misc/update-button/__tests__/update-button-with-delay.test.tsx:63]: Is this meant to be hasUpdate to be true?

File: packages/ts-newskit/src/slices/lead-story-1/__tests__/article-stack.test.tsx [hunk 4]
```diff
@@ -17,23 +17,29 @@ const mockClickHandler = jest.fn();
...
+    switch (breakpoint) {
+      case 'lg':
+        return '1024px';
+
+      case 'xl':
+        return '1440px';
+      default:
+        return '768px';
+    }
+  };
```
[the remark this answers is not in the document]
Comment (yukiinasawa) [reply packages/ts-newskit/src/slices/lead-story-1/__tests__/article-stack.test.tsx:31]: Maybe worth waiting for https://github.com/newsuk/cps-content-render/pull/6716 to marge in, then we could check with `width: 100%` here?

File: packages/ts-newskit/src/components/misc/update-button/__tests__/update-button-with-delay.test.tsx (hunk 3 above)
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/misc/update-button/__tests__/update-button-with-delay.test.tsx]: just pushed a commit to make it clearer

**Review by bibicollins1 — APPROVED** [review]

**Review by douglasmik — APPROVED** [review]

**Review by yukiinasawa — APPROVED** [review]

**Pull request merged**